bunq/sdk/context.py

```diff
@@ -254,33 +254,46 @@ def session_context(self):
 
         return self._session_context
 
-    def save(self, path=None):
+    def save(self, path=None, to_json=False):
         """
         :type path: str
+        :type to_json: bool
 
-        :rtype: None
+        :rtype: Union[None, str]
         """
 
         if path is None:
             path = self._PATH_API_CONTEXT_DEFAULT
 
+        if to_json:
+            return converter.class_to_json(self)
+
         with open(path, self._FILE_MODE_WRITE) as file:
             file.write(converter.class_to_json(self))
 
     @classmethod
-    def restore(cls, path=None):
+    def restore(cls, path=None, json_data=None):
         """
         :type path: str
+        :type json_data: str
 
         :rtype: ApiContext
         """
 
         if path is None:
             path = cls._PATH_API_CONTEXT_DEFAULT
 
+        if json_data is not None:
+            return converter.json_to_class(ApiContext, json_data)
+
         with open(path, cls._FILE_MODE_READ) as file:
             return converter.json_to_class(ApiContext, file.read())
 
+    def __eq__(self, other):
+        return self.token == other.token \
```

Comment:
@PJUllrich I might have the wrong indent there; probably adding one tab on the left for each of the indented lines will fix it? :)